build/lib/src/generate/expected_outputs.dart

```diff
@@ -42,59 +42,74 @@ extension on AssetId {
 }
 
 abstract class _ParsedBuildOutputs {
-  static final RegExp _captureGroup = RegExp('{{}}');
+  static final RegExp _captureGroup = RegExp('{{([\\w\\d]*)}}');
```

Comment:
```suggestion
  static final RegExp _captureGroup = RegExp('{{(\\w*)}}');
```

word characters already include 0-9 :)